tests/FormBuilderTestCase.php

```
@@ -184,7 +184,9 @@ public function setUp(): void
     {
         parent::setUp();
 
-        $this->withoutDeprecationHandling();
+        // $this->withoutDeprecationHandling();
```

Why? Deprecations are errors we should find, not silently ignore.